scripts/msmarco_v2_doc_segment.py

```diff
@@ -0,0 +1,90 @@
+import argparse
```

Comment:
should we move into `scripts/msmarco_v2` and call something like `segment_docs.py`?

I like script to begin with verbs...